### PR TITLE
Add events to the calculator

### PR DESCRIPTION
**Pull request by cyrilmuchemi**
### Project requirements

To help you concentrate on how to work with React events, we have developed the math logic you need to make the Calculator component work. You can use the files [calculate.js](./code-samples/calculate.js) and [operate.js](./code-samples/operate.js).

- Copy  the files `calculate.js` and `operate.js` into a `logic/` directory in your project.
- Analyze the files `calculate.js` and `operate.js`, be sure you understand what they do (note: you need to add [big.js](https://www.npmjs.com/package/big-js) as dependency).
- Import the files in your Calculator component.
- Implement the event handlers you need to use the math logic from `calculate.js` and `operate.js` in your React component.
- Practice what you have learned until now: use `state` and `props`, pass props to *child* components, use lifecycle methods, *lift state up*.
- Test your app in the browser. It should be able to perform math operations.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,40 +1,57 @@
...
   render() {
+    const { next, operation, total } = this.state;
     return (
       <div className="Calculator">
         <div className="display">
-          <span>0</span>
+          <span>{total}</span>
+          <span>{operation}</span>
+          <span>{next}</span>
         </div>
```
Comment (brytebee) [inline src/components/Calculator.js:26]: - [ ] **Required:** Your events work perfectly, however, the `span`s seems to be messing with the positioning of your component, kindly fix this and ensure it's stable. Request another review of your dashboard.

image

Comment (ShadyShawkat) [inline src/components/Calculator.js:26]: - [ ] The design still looks off. Kindly fix the calculator display as instructed by the previous reviewer.
image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @cyrilmuchemi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Image

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @cyrilmuchemi, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**